pkg/controller/migstorage/validation.go

```diff
@@ -105,55 +93,60 @@ func (r ReconcileMigStorage) validateBackupStorage(storage *migapi.MigStorage) e
 			Status:   True,
 			Reason:   NotSet,
 			Category: Critical,
-			Message:  InvalidBSCredsSecretRefMessage,
+			Message:  "spec.BackupStorageConfig.CredsSecretRef name or namespace is empty",
 		})
 		return nil
 	}
 
 	// Secret
 	secret, err := storage.GetBackupStorageCredSecret(r)
 	if err != nil {
-		return liberr.Wrap(err)
+		return liberr.Wrap(fmt.Errorf("error getting migstorage backup provider secret %s/%s: %v",
+			storage.Spec.BackupStorageConfig.CredsSecretRef.Namespace,
+			storage.Spec.BackupStorageConfig.CredsSecretRef.Name,
+			err))
```

Comment:
This is an unanticipated/unrecoverable error which is usually just logged as a stack trace.
I'm a little concerned about rewriting errors.  The fmt.Errorf() probably returns a new error that will not behave as expected for an errors.Is() test.
For this kind of failure, I don't see how the namespace and name is useful.

Reply:
While I do see how the namespace and name can be useful, since this is changing the behavior, I drop this change in favor of the original change anyway